x-pack/legacy/plugins/uptime/server/lib/adapters/pings/elasticsearch_pings_adapter.ts

```diff
@@ -262,15 +221,11 @@ export class ElasticsearchPingsAdapter implements UMPingsAdapter {
       histogram,
       interval,
     };
-  }
+  },
 
-  /**
-   * Count the number of documents in heartbeat indices
-   * @param request Kibana server request
-   */
-  public async getDocCount(request: any): Promise<DocCount> {
-    const { count } = await this.database.count(request, { index: INDEX_NAMES.HEARTBEAT });
+  getDocCount: async ({ callES }) => {
+    const { count } = await callES('count', { index: INDEX_NAMES.HEARTBEAT });
 
     return { count };
-  }
-}
+  },
+};
```

Comment:
Again file naming, unrelated to this PR, elasticsearch_pings_adapter.ts should be just elasticsearch_pings, since it's inside adapters folder, so it doesn't make sense to prefix/postfix files. It's a twitter rules as well, when you have already used "lol", "haha" can't be used in the same tweet :P

Reply:
Lol, I see what you're saying haha.

Addressed in https://github.com/elastic/kibana/pull/51403#discussion_r357282576.